eng/common/performance/perfhelixpublish.proj

```diff
@@ -64,6 +64,14 @@
     <WorkItemCommand>$(WorkItemCommand) $(CliArguments)</WorkItemCommand>
   </PropertyGroup>
 
+  <PropertyGroup Condition="$(HelixSourcePrefix) == 'pr'">
```

Comment:
I think you need to also escape `$(HelixSourcePrefix)` for the comparison to be correct:

```suggestion
  <PropertyGroup Condition="'$(HelixSourcePrefix)' == 'pr'">
```